### PR TITLE
Issue kubelet-api cert in nodeup

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: pkg/bundle/builder.go [hunk 1]
```diff
@@ -239,37 +239,18 @@ func (b *Builder) buildPKIFiles(cluster *kops.Cluster, ig *kops.InstanceGroup, k
 	keys = append(keys, "kube-proxy")
 
 	if ig.IsMaster() {
```
Comment (johngmyers) [inline pkg/bundle/builder.go:241]: The fact that kubelet-api was not in here to be removed suggests that baremetal is not being used.

Comment (hakman) [reply pkg/bundle/builder.go:241]: I think @olemarkus was thinking of removing baremetal.

Comment (johngmyers) [reply pkg/bundle/builder.go:241]: Filed #9360

**Comment by justinsb** [comment]
Thanks @johngmyers 

/approve
/lgtm

**Pull request merged**